src/types.ts

```diff
@@ -5,3 +5,9 @@ export interface ColumnConfiguration {
     hide?: boolean;
     //TODO: filters
 }
+
+export const FILTER_DATA_TYPE = {
+    string: "string",
+    numeric_range:"numeric_range",
+    string_range:"string_range"
+};
```

Comment:
Were you trying to implement an [enum?](https://www.typescriptlang.org/docs/handbook/enums.html)

Reply:
enum is perfect for this case, I was looking for something like that. Thanks.